packages/luthor/lib/src/validators/string_validator.dart

```diff
@@ -82,6 +83,13 @@ class StringValidator extends Validator {
     return this;
   }
 
+  /// Validates that the string is an ip with optional parameter for IPv4 and IPv6 as "v4" and "v6" respectively.
+  StringValidator ip({String? message, String? version}) {
```

Comment:
Can you make this an enum value? For example: `IpVersion.v4`. I'd like not to have magic strings and have everything as typesafe as possible

Reply:
sure, no issue